### PR TITLE
LPS-197512 Get the label using current locale before using site default locale on LabelMap

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-197512

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowNodeTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		defaultWorkflowNode.setLabelMap(
+			HashMapBuilder.put(
+				LocaleUtil.BRAZIL, "Revisar Regra de Negócio"
+			).build());
+		defaultWorkflowNode.setName("businessRuleReview");
+
+		Assert.assertEquals(
+			"Revisar Regra de Negócio",
+			defaultWorkflowNode.getLabel(LocaleUtil.BRAZIL));
+	}
```
Comment (MarinhoFeliphe) [inline portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowNodeTest.java:74]: Hey @p-albuquerque could you try use comments to divide those test cases?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hey @p-albuquerque please see my comments,

Nice job!

File: portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowNodeTest.java (hunk 1 above)
Comment (p-albuquerque) [reply portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowNodeTest.java:74]: The same comments that we had been agreed before [here](https://github.com/liferay-workflow/liferay-portal/pull/1032/commits/be06a307d58d3c9a2fc9f8b5c6dbc546227bb288)?

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by p-albuquerque** [comment]
ci:forward:force

**Comment by p-albuquerque** [comment]
Directly sent: https://github.com/brianchandotcom/liferay-portal/pull/145947

**Pull request closed without merging**